src/EditorFeatures/Core.Wpf/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -248,7 +250,8 @@ public InlineRenameFileRenameInfo GetFileRenameInfo()
                         return InlineRenameFileRenameInfo.TypeWithMultipleLocations;
                     }
 
-                    if (OriginalNameMatches(_document, RenameSymbol.Name))
+                    var symbolSourceDocument = _document.Project.Solution.GetDocument(RenameSymbol.Locations.Single().SourceTree);
+                    if (symbolSourceDocument is object && OriginalNameMatches(symbolSourceDocument, RenameSymbol.Name))
```

Comment:
1. `is object` feels much less clear than `!= null`.
2. it feels like OriginalNameMatches could do this check.

Reply:
also... i don't know why we ahve a helper method here.

Reply:
changed to use helper that was added with `RenameDocumentAsync`